man/coredump.conf.xml

```diff
@@ -112,13 +112,25 @@
       <varlistentry>
         <term><varname>EnterNamespace=</varname></term>
 
-        <listitem><para>Controls whether <command>systemd-coredump</command> will attempt to use the mount tree of
-        a process that crashed in PID namespace. Access to the namespace's mount tree might be necessary to generate
-        a fully symbolized backtrace. If set to <literal>yes</literal>, then <command>systemd-coredump</command> will
-        obtain the mount tree from corresponding mount namespace and will try to generate the stack trace using the
-        binary and libraries from the mount namespace. Note that the coredump of the namespaced process might
-        still be saved in <filename>/var/lib/systemd/coredump/</filename> even if <varname>EnterNamespace=</varname>
-        is set to <literal>no</literal>. Defaults to <literal>no</literal>.</para>
+        <listitem><para>For processes belonging to a PID namespace, controls whether
+        <command>systemd-coredump</command> shall attempt to process core dumps on the host, using debug
+        information from the namespace of the process that crashed. Access to the
+        namespace's mount tree might be necessary to generate a fully symbolized backtrace. If set to
```

Comment:
The wording here is inhertited from the old version, where it was sloppy too. "mount tree of the process that crashed in a PID namespace" is correct. "namespace's mount tree" and "information from the namespace of the process that crashed" is … meh, because we have both a PID namespace and a mount namespace and we need to be careful when talking about "a namespace". Let's be precise and use the first formulation.